pkgs/development/compilers/elm/packages/elm-review/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+}:
+
+buildNpmPackage rec {
+  pname = "elm-review";
+  version = "2.12.0";
+
+  src = fetchFromGitHub {
+    owner = "jfmengels";
+    repo = "node-elm-review";
+    rev = "v${version}";
+    hash = "sha256-+RF1D1gTsZlxE+Pe29T11TT6bWkt17bX7MX0YjBUPcI=";
+  };
+
+  npmDepsHash = "sha256-suNOrGhHoAopBnIar8+HNUk4xwDtFq1gj3GvN2tD5l0=";
+
+  postPatch = ''
+    sed -i '/elm-tooling install/d' package.json
```

Comment:
This leaves the `"scripts"` object with a trailing comma, which isn't allowed in JSON.

Reply:
good catch. definitely it must be these ad hoc patches which do string manipulations.